src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -37,23 +37,41 @@
 #include "utils/macro.h"
 #include "utils/make_shared.h"
 #include "application_manager/policies/policy_handler.h"
-#include "application_manager/test/resumption/include/application_mock.h"
+#include "application_manager/test/include/application_manager/application_mock.h"
 #include "utils/custom_string.h"
-#include "policy/mock_policy_settings.h"
 #include "application_manager/policies/policy_handler.h"
-#include "application_manager/mock_application_manager.h"
+#include "application_manager/test/include/application_manager/mock_application.h"
+#include "application_manager/test/include/application_manager/mock_application_manager.h"
 #include "application_manager/event_engine/event_dispatcher.h"
 #include "application_manager/state_controller.h"
 #include "application_manager/resumption/resume_ctrl.h"
-
+#include "policy/mock_policy_settings.h"
+#include "application_manager/policies/mock_policy_handler_interface.h"
 namespace application_manager {
 namespace test {
 
+namespace {
+const uint32_t kCorrelationId = 1;
+const uint32_t kAppId = 123;
+#ifdef OS_WINDOWS
+const std::string kFullFilePath = "C:\\file.txt";
+const std::string kRelativeFilePath = "file.txt";
+const std::string kAppStorageFolder = "";
```

Comment:
@indlu By default `std::string` is initialized as empty, so this assignment is not needed.